packages/@aws-cdk/aws-codepipeline-api/lib/action.ts

```diff
@@ -36,6 +36,32 @@ export function defaultBounds(): ActionArtifactBounds {
   };
 }
 
+/**
+ * The interface used in the {@link Action#bind()} callback.
+ */
+export interface IBindInfo {
```

Comment:
I believe this can be a struct (`BindInfo`) instead of an interface. It's basically a collection of fields, no? We don't expect anyone to implement this interface.